test/LibraryManager.IntegrationTest/LibmanTestsUtility.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.Test.Apex.Editor;
+using Microsoft.Test.Apex.VisualStudio.Editor;
+using Omni.Common;
+
+namespace Microsoft.Web.LibraryManager.IntegrationTest
+{
+    public class LibmanTestsUtility
+    {
+        public static string WaitForCompletionEntries(IVisualStudioTextEditorTestExtension editor, IEnumerable<string> expectedCompletionEntries, bool caseInsensitive, int timeout)
```

Comment:
Seems a bit odd to use a string to indicate failure. Can you explain why you think this is better than throwing an exception as the other code does?

Reply:
As string compare cost more, I would make a change here as Editors do.